geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -508,6 +517,10 @@ public JMXConnectorServer getJmxConnectorServer() {
     return jmxConnectorServer;
   }
 
+  public void setJmxConnectorServer(JMXConnectorServer jmxConnectorServer) {
+    this.jmxConnectorServer = jmxConnectorServer;
+  }
+
```

Comment:
Should this method be `private` instead of `public`?

Reply:
This method is added for the test. So it should be non-private. We can remove the `public` scope.